camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/mapred/EtlInputFormat.java

```diff
@@ -527,6 +533,39 @@ private void writeRequests(List<CamusRequest> requests, JobContext context) thro
     return offsetKeysMap;
   }
 
+  public PartitionMetadata refreshPartitionMetadataOnLeaderNotAvailable(PartitionMetadata partitionMetadata,
+      TopicMetadata topicMetadata, JobContext context, int retryTimes) throws InterruptedException {
```

Comment:
"retryTimes" is ambiguous. Does it include or exclude the first try? Better to use "tryTimes" or "numTries"